modules/deploy/pages/deployment-option/cloud/gcp-private-service-connect.adoc

```diff
@@ -0,0 +1,425 @@
+            "agent_service_account": { "email": "$BYOVPC_AGENT_SERVICE_ACC_EMAIL" },
+            "connector_service_account": { "email": "$BYOVPC_CONNECTORS_SERVICE_ACC_EMAIL" },
+            "console_service_account": { "email": "$BYOVPC_CONSOLE_SERVICE_ACC_EMAIL" },
+            "redpanda_cluster_service_account": { "email": "$BYOVPC_REDPANDA_SERVICE_ACC_EMAIL" },
+            "gke_service_account": { "email": "$BYOVPC_GKE_SERVICE_ACC_EMAIL" },
+            "tiered_storage_bucket": { "name" : "$BYOVPC_TIERED_STORAGE_BUCKET" },
+        }
+    }
+}
+EOF`
+
+curl -vv -X POST \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_POST_BODY" $PUBLIC_API_ENDPOINT/v1beta1/clusters
+----
+
+== Enable PSC on an existing BYOVPC cluster
+
+. Update the Redpanda Cloud Agent IAM role.
++
+To allow the agent to create and manage the service attachment, add the following permissions to its IAM role:
++
+[,bash]
+----
+compute.forwardingRules.use
+compute.regionOperations.get
+compute.serviceAttachments.create
+compute.serviceAttachments.delete
+compute.serviceAttachments.get
+compute.serviceAttachments.list
+compute.serviceAttachments.update
+compute.subnetworks.use
+----
+
+. In the Redpanda Cloud UI, go to the cluster overview and copy the cluster ID from the **Details** section.
++
+[,bash]
+----
+CLUSTER_ID=<cluster_id>
+----
+
+. Make a https://docs.api.redpanda.com/#patch-/v1beta1/clusters/-cluster.id-[`PATCH /v1beta1/clusters/{cluster.id}`] request to update the cluster to include the newly-created PSC NAT subnet and enable PSC.
++
+Set the following environment variables:
++
+--
+* `ACCEPT_LIST`: a JSON list specifying the projects from which incoming connections will be accepted. All other sources. For example, `[{"source": "consumer-project-ID-1"},{"source": "consumer-project-ID-2"}]`.
+* `PSC_NAT_SUBNET_NAME`: The name of the PSC NAT subnet. Use the fully-qualified name, for example `"projects/<PROJECT>/regions/<REGION>/subnetworks/<SUBNET-NAME>"`.
+--
++
+[,bash]
+----
+ACCEPT_LIST='[]'
+PSC_NAT_SUBNET_NAME='<PSC NAT subnet name>'
+CLUSTER_PATCH_BODY=`cat << EOF
+{
+    "customer_managed_resources": {
+    "gcp": {
+        "psc_nat_subnet_name": "$PSC_NAT_SUBNET_NAME"
+    }
+    },
+    "private_link": {
+    "enabled": true,
+    "gcp": {
+        "consumer_accept_list": $ACCEPT_LIST
+    }
+    }
+}
+EOF`
+curl -v -X PATCH \
+-H "Content-Type: application/json" \
+-H "Authorization: Bearer $AUTH_TOKEN" \
+-d "$CLUSTER_PATCH_BODY" ${PUBLIC_API_ENDPOINT}/v1beta1/clusters/$CLUSTER_ID
+----
++
+Wait for the cluster to apply the new configuration (around 15 minutes). The PSC service attachment is available when the cluster update is complete. You can monitor the service attachment creation by running the following `gcloud` command:
++
+[,bash]
+----
+gcloud compute service-attachments list --project '<service-project-ID>'
+----
+
+== Optional: Deploy consumer resources
```

Comment:
Can we explain why these tasks are optional and under what circumstances a user might want to carry them out?

Reply:
I think we should remove this part altogether. It relies on TF modules I didn't know we have no way of distributing to our customers. We can provide `gcloud` commands instead.